src/test/resources/configFile.properties

```diff
@@ -0,0 +1,16 @@
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project.
+TRUSTSTOREPATH=src/test/resources/localhost.truststore.p12
+KEYSTOREPATH=src/test/resources/localhost.keystore.p12
+TRUSTSTOREPASSWORD=password
```

Comment:
password shouldn't be kept as a plain text. Instead, encrypted form should be placed here. It needs to be decrypted in the code.  It's a security issue.